tq/transfer_queue.go

```diff
@@ -768,7 +768,7 @@ func (q *TransferQueue) Wait() {
 		close(watcher)
 	}
 
-	q.meter.Finish()
+	q.meter.Flush()
```

Comment:
I don't think that `Flush()` will call `close(m.updates)` which means that the `*tasklog.Logger` won't yield to future `Task` implementations.

Reply:
EDIT: Re-reading your PR body makes this make sense to me. Is the update channel closed elsewhere, though?

Reply:
Yup, in `ReportErrors()`: https://github.com/git-lfs/git-lfs/pull/2806/files#diff-3cff4ed9c5b60b5b81240c88cd74e404R263